### ISSUE TITLE
Are the types correct on the latest version?

### ISSUE THREAD

**Issue by matt-dalton** [mixpanel/mixpanel-react-native#83]
I've changed my initialisation for the latest version:
```
const mixpanel = new _Mixpanel(config.MIXPANEL_TOKEN)
mixpanel.init()
```
But now I'm getting a Typescript error on the first line
```
Expected 0 arguments, but got 1.

24         const mixpanel = new _Mixpanel(config.MIXPANEL_TOKEN)
                                          ~~~~~~~~~~~~~~~~~~~~
Found 1 error.
```

Is there a workaround/fix for this? Or am I doing something wrong?

**Comment by byeokim** [comment]
[`index.d.ts`](https://github.com/mixpanel/mixpanel-react-native/blob/9e740d66c63a79c62643db9f5e4a13cdbe9f7a2e/index.d.ts#L4)'s `Mixpanel` doesn't have `constructor(token: string)` type definition. I think you could insert it to your project's `node_modules/mixpanel-react-native/index.d.ts` by yourself as a workaround. Or should I create a pull request for this @zihejia?

**Comment by zihejia** [comment]
@matt-dalton Thanks for bringing it up.
hi @byeokim , You are more than welcome to submit a PR. Thank you so much!

**Comment by zihejia** [comment]
It's fixed in v1.3.2. Thanks so much @byeokim !
